data/campaigns/atl02.wmf/scripting/tribes/init.lua

```diff
@@ -0,0 +1,64 @@
+include "tribes/scripting/help/time_strings.lua"
+
+push_textdomain("scenario_atl02.wmf")
+
+local result = {
+   atlanteans = {
+      buildings = {
+         {
+            name = "atlanteans_trading_post",
+            helptexts = {
+               -- TRANSLATORS: Purpose helptext for an atlantean market site: Trading post
+               purpose = pgettext("atlanteans_building", "Enables the share of goods with other players.")
+            }
+         },
+         {
+            name = "atlanteans_scouts_house1",
+            helptexts = {
+               -- TRANSLATORS: Purpose helptext for an atlantean production site: Scout's House
+               purpose = pgettext("atlanteans_building", "Explores unknown territory.")
+            }
+         },
+      },
+      workers_order = {
+         {
+            { name = "atlanteans_trader" }
```

Comment:
Helptext missing, same for the wares below

Reply:
the wares are just a bloody workaround to count whether a trade (and which one) has been carried out. Any ideas to solve this in a way without the wares would be welcome.